deploy/docker/metricbeat.docker.yml

```diff
@@ -0,0 +1,34 @@
+metricbeat.config:
+  modules:
+    path: ${path.config}/modules.d/*.yml
+    # Reload module configs as they change:
+    reload.enabled: false
+
+# To enable hints based autodiscover, remove `metricbeat.config.inputs` configuration and uncomment this:
```

Comment:
probably you want to remove this comment

Reply:
will do.